src/PowerShellEditorServices.Channel.WebSocket/PowerShellEditorServices.Channel.WebSocket.csproj

```diff
@@ -1,20 +1,13 @@
-﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), PowerShellEditorServices.Common.props))\PowerShellEditorServices.Common.props" />
 
   <PropertyGroup>
     <AssemblyTitle>PowerShell Editor Services WebSocket Protocol Channel</AssemblyTitle>
-    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
+    <TargetFramework>net451</TargetFramework>
```

Comment:
We want to use it in Azure to provide a better PowerShell editing experience, due to the complexity of Azure Resource Manager's architecture, there's no way to maintain a persistent WebSocket connection between a browser and a backend server.  Others have been using long-polling for "connected" sessions so I figured it might be better to make that the primary approach.